src/index.html

```diff
@@ -5,9 +5,76 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    >
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+
+      <img
+        src="./images/imac.jpeg"
+        alt="product-image"
+        class="card__img"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="card__rang">
+
+        <div class="stars stars--4">
+          <div class="stars__element"></div>
+          <div class="stars__element"></div>
+          <div class="stars__element"></div>
+          <div class="stars__element"></div>
+          <div class="stars__element"></div>
+        </div>
+
+        <p class="card__reviews">
+          Reviews: 5
+        </p>
+
+      </div>
+
+      <div class="card__expense">
+        <p class="card__expense-price">
+          Price:
+        </p>
+
+        <p class="card__expense-cost">
+          $2,199
+        </p>
+      </div>
+
+      <button
+        type="button"
+        class="button"
```

Comment:
button is also an element of the card. use proper class according to BEM

Reply:
deliberately named it button so that it can be reused